lib/tokens/CookieTokenManager.js

```diff
@@ -26,28 +26,48 @@ class CookieTokenManager extends TokenManager {
 
   _getToken(callback) {
     debug('Submitting cookie token request.');
+    let boundedCompleted = this._reqCompleted.bind({
+      cb: callback
+    });
+
+    let protocol;
+    if (this._sessionUrl.startsWith('https')) {
+      protocol = require('https');
+    } else {
+      protocol = require('http');
+    }
+    let tokenAgent = new protocol.Agent({
+      keepAlive: false,
+      maxSockets: 3
```

Comment:
This agent is only used once at any given time for a cookie request, I think we may as well use `maxSockets: 1` wdyt?

Reply:
Changed in 00eccec

Yes, correct. I couldn't dare to set it to 1, because it is always good if there is a backup(+1). But in this case, it is not valid. Thanks for the encouragement.